lib/openstudio-standards/prototypes/common/objects/Prototype.Model.exterior_lights.rb

```diff
@@ -47,37 +67,61 @@ def model_add_typical_exterior_lights(model, exterior_lighting_zone_number, onsi
+
+    occupancy_setback_reduction = 0.0
+    ext_lights_sch_other = model.alwaysOnDiscreteSchedule
+    if exterior_lighting_properties.has_key?('occupancy_setback_reduction')
+      if !exterior_lighting_properties['occupancy_setback_reduction'].nil? && (exterior_lighting_properties['occupancy_setback_reduction'] > 0.0)
+        occupancy_setback_reduction = exterior_lighting_properties['occupancy_setback_reduction']
+      end
+    else
+      unless ['90.1-2004', '90.1-2007', 'DOE Ref Pre-1980', 'DOE Ref 1980-2004', 'DEER 1985', 'DEER 1996', 'DEER 2003', 'DEER 2007', 'DEER Pre-1975'].include?(template)
+        occupancy_setback_reduction = 0.3
+      end
+    end
+    if occupancy_setback_reduction > 0
       ext_lights_sch_other = OpenStudio::Model::ScheduleRuleset.new(model)
-      setback_value = 1.0 - exterior_lighting_properties['occupancy_setback_reduction']
+      setback_value = 1.0 - occupancy_setback_reduction
       default_day = ext_lights_sch_other.defaultDaySchedule
       default_day.addValue(OpenStudio::Time.new(0, end_setback_shutoff[:hr], end_setback_shutoff[:min], 0), setback_value)
       default_day.addValue(OpenStudio::Time.new(0, start_setback_shutoff[:hr], start_setback_shutoff[:min], 0), 1.0)
-      OpenStudio.logFree(OpenStudio::Info, 'openstudio.prototype.exterior_lights', "Non Facade and Landscape lights reduce by #{exterior_lighting_properties['occupancy_setback_reduction'] * 100} % from #{start_setback_shutoff} to #{end_setback_shutoff}")
-    else
-      ext_lights_sch_other = model.alwaysOnDiscreteSchedule
+      OpenStudio.logFree(OpenStudio::Info, 'openstudio.prototype.exterior_lights', "Non Facade and Landscape lights reduce by #{occupancy_setback_reduction * 100} % from #{start_setback_shutoff} to #{end_setback_shutoff}")
     end
 
     # add exterior lights for parking area
     if !area_length_count_hash[:parking_area_and_drives_area].nil? && area_length_count_hash[:parking_area_and_drives_area] > 0
-
       # lighting values
       multiplier = area_length_count_hash[:parking_area_and_drives_area] * onsite_parking_fraction
-      power = exterior_lighting_properties['parking_areas_and_drives']
       name_prefix = 'Parking Areas and Drives'
+      ext_lights_def_name = "#{name_prefix} Def (W/ft^2)"
+      if exterior_lighting_properties.has_key?('parking_areas_and_drives')
+        power = exterior_lighting_properties['parking_areas_and_drives']
+      else
+        search_criteria = {
+          'template' => template,
+          'lighting_zone' => exterior_lighting_zone_number,
+          'allowance_type' => 'parking lots and drives'
+        }
+        exterior_lighting_parking_req = standards_lookup_table_first(table_name: 'exterior_lighting', search_criteria: search_criteria)
+        power = exterior_lighting_parking_req['allowance']
+        if exterior_lighting_parking_req['allowance_unit'] == 'W'
```

Comment:
in what instance is the allowance in watts?

Reply:
One example is the base allowance.

Reply:
Makes sense for base allowance. It seems like this switch statement applies to all lighting types. A lot of duplicate code. It's not necessary in parking and most others, because all standard entries are in W/ft2. I can handle the cleanup unless there is a way you'd prefer to do it.

Reply:
I removed the statements and also clarified the names of the definitions to be more in line with the inputs.